src/components/balance/PlmBalance.vue

```diff
@@ -142,17 +161,40 @@
           </p>
         </div>
       </div>
+
+      <div
+        v-if="isCheckMetaMask && isAstar"
```

Comment:
Is there a reason for checking only for Astar?

Reply:
I understand that the 'Switch to Plasm Lockdrop account' button is required for Astar network only. Please advise if the button is required for Shiden network as well.
![image](https://user-images.githubusercontent.com/92044428/149827277-6877b004-cc25-4035-9004-868ee7303c77.png)

Reply:
Shiden users will need this too since their rewards are transferred to that account too.
Plus, this feature is not network-dependent. As long as the chain supports custom signature pallet, it should be available.